kommandant.gemspec

```diff
@@ -18,4 +18,7 @@ Gem::Specification.new do |spec|
   end
 
   spec.add_dependency "rails", ">= 7.0.5"
```

Comment:
Kan vi ikke nøjes med turbo-rails? eller er det for at sikre at man kommer over Rails 7?

Reply:
Det er en engine, så det tror jeg ikke går.